src/Console/PrefabCommand.php

```diff
@@ -351,33 +354,6 @@ protected function prefabModule(string $module): void
 
         // Todo: extract merge composer.json
 
-        $after = <<< 'AFTER'
```

Comment:
hier heb je verwijderd dat helpers wordt toegvoegd aan composer.json (?)

Reply:
Er was een probleem met helpers toevoegen, maar dat heb ik al opgelost in de `update-main` branch